tests/forms.py

```diff
@@ -179,13 +179,13 @@ def clean(self):
 
 class RaisesErrorCustomCleanMultiform(UserProfileMultiForm):
     def clean(self):
-        cleaned_data = super(UserProfileMultiForm, self).clean()
+        cleaned_data = super().clean()
```

Comment:
This super call should be left  as was: `super(UserProfileMultiForm, self).clean()`.
Notice that the original code uses `UserProfileMultiForm` as the first argument to `super` even though the class in which the method is defined is `RaisesErrorCustomCleanMultiform`. This is not the same as the behavior of `super()` when the arguments are omitted.